cartoframes/core/managers/context_manager.py

```diff
@@ -307,45 +301,18 @@ def _create_auth_client(credentials, public=False):
     )
 
 
-def _rows(df, dataframe_columns_info):
+def _compute_copy_data(df, dataframe_columns_info):
     for index, _ in df.iterrows():
         row_data = []
-        for c in dataframe_columns_info.columns:
-            col = c.dataframe
-            if col not in df.columns:
-                if df.index.name and col == df.index.name:
-                    val = index
-                else:  # we could have filtered columns in the df. See DataframeColumnsInfo
-                    continue
-            else:
-                val = df.at[index, col]
+        for column in dataframe_columns_info.columns:
+            val = df.at[index, column.name]
```

Comment:
I think we will have a bug here in case we normalize a column name in `DataframeColumnInfo` (the name in the geoDataFrame and the name here will be different)

Reply:
Yes. You are right! I'll update this part

Reply:
Fixed in https://github.com/CartoDB/cartoframes/pull/1265/commits/439abbca49f8e0750719925791658884168f3d9c